lib/applicants/utils.ts

```diff
@@ -21,3 +22,25 @@ export const getActivePermit = async (applicantId: number): Promise<Permit | nul
 
   return activePermits.length > 0 ? activePermits[0] : null;
 };
+
+/**
+ * Get the most recent permit of an applicant
+ * @param applicantId ID of the applicant
+ * @returns The most recent permit of the applicant
+ */
+export const getMostRecentPermit = async (applicantId: number): Promise<Permit> => {
+  const permit = await prisma.applicant
+    .findUnique({
+      where: { id: applicantId },
+    })
+    .permits({
+      orderBy: { createdAt: SortOrder.DESC },
+      take: 1,
+    });
+
+  if (!permit) {
```

Comment:
```suggestion
  if (permit.length === 0) {
```
Careful here, in JS, `[]` is truthy.